spec/registrant_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+  it 'exists' do
+    registrant = Registrant.new("x",3)
+    expect(registrant).to be_an_instance_of(Registrant)
+  end
+
+  it "has atttibutes" do
+    registrant = Registrant.new('Bruce', 18, true )
+    expect(registrant.name).to eq("Bruce")
+    expect(registrant.age).to eq(18)
+    expect(registrant.permit?).to eq(true)
+  end
+
+  it "can be updated" do
```

Comment:
I'd try to be more specific with these tests. This one could be `it "can earn permit"` and then you just test that the earn_permit method changes the value of permit?